### DISCUSSION TITLE
Can I take back my account after violating the terms of service?

### DISCUSSION THREAD

**Discussion by HosseinMohammadi57** [community/community#137267] [New to GitHub] [answered]
Hi,
I am a student, which used github for hosting his persional website. I found that my account has been suspended. The only reason that I think for this decision is that I used a html5 template, and deleted the designer's name and address. Does it conflicts with the terms of service? If yes, Can I take back my account? I just want to store my stuff in github and have a minimal website for myself.

**Comment by MiguelCardoso19** [comment, marked as answer]
Yes, removing the designer’s name might have caused the suspension. Contact GitHub Support(https://support.github.com/request), fix the attribution, and they might reinstate your account.
